File: pset1/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>




int main(void) {

    int number;


    do{
     number = get_int("Height: ");

    }
    while (number < 1 || number > 8);

   for ( int i = 0; i < number; i++) {

       for ( int j = 0; j < number ; j++){

          if (i + j < number - 1)
              printf(" ");
          else
              printf("#");

       }

      printf("\n");
   }



}

File: pset6/mario/less/mario.py
from cs50 import get_int

n = get_int("Height: ")

if n > 1 and n < 9:
    for i in range(n):
        for j in range(n):
            if i + j < n - 1:
                print(" ", end = "")
            else:
                print("#", end = "")

        print()

else:
    n = get_int("Height: ")
